packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -1,40 +1,6 @@
 <Project>
-  <PropertyGroup>
-    <_MicrosoftNetSdkWindowsDesktop>true</_MicrosoftNetSdkWindowsDesktop>
-    <EnableDefaultPageItems Condition="'$(EnableDefaultPageItems)' == ''">true</EnableDefaultPageItems> 
-    <EnableDefaultApplicationDefinition Condition="'$(EnableDefaultApplicationDefinition)' == ''">true</EnableDefaultApplicationDefinition>
-  </PropertyGroup>
 
-  <PropertyGroup>
-    <!-- 
-    WindowsDesktop SDK supports WPF and WinForms on 
-      - .NET Core 3.0 and greater
-      - .NET Framework 3.0 and greater
-   
-    Note that on .NET Framework versions < 4.0, additional workarounds may be required to build applications 
-    using the SDK style projects. For e.g., see https://github.com/microsoft/msbuild/issues/1333
-    
-    Irrespective of whether '$(TargetFrameworkIdentifier)' is '.NETCoreApp' or '.NETFramework', 
-    the minimum value of $(_TargetFrameworkVersionValue) we will be testing for is '3.0'
-    
-  -->
-    <_WindowsDesktopSdkTargetFrameworkVersionFloor>3.0</_WindowsDesktopSdkTargetFrameworkVersionFloor>
-
-    <!-- 
-      Represents an undefined TFV value. This will be used in comparisons of _TargetFrameworkVersionValue (defined in Microsoft.NET.WindowsDesktop.targets)
-      to identify when a TFV is undefined
-    -->
-    <_UndefinedTargetFrameworkVersion>0.0</_UndefinedTargetFrameworkVersion>
-    
-    <!--
-      Initial/Default value set to 'undefined'. Updated in Microsoft.NET.WindowsDesktop.targets
-    --> 
-    <_TargetFrameworkVersionValue>$(_UndefinedTargetFrameworkVersion)</_TargetFrameworkVersionValue>
-  </PropertyGroup>
-
-  <ItemGroup Condition=" ('$(EnableDefaultItems)' == 'true') And ('$(UseWPF)' == 'true') And 
-                         ('$(_TargetFrameworkVersionValue)' != '$(_UndefinedTargetFrameworkVersion)') And 
-                         ('$(_TargetFrameworkVersionValue)' >= '$(_WindowsDesktopSdkTargetFrameworkVersionFloor)')">
+  <ItemGroup Condition=" '$(_EnableWindowsDesktopGlobbing)' == 'true' ">
```

Comment:
Extract a new property in target to reduce the surface area, since props will always be imported